### DISCUSSION TITLE
What is depName?

### DISCUSSION THREAD

**Discussion by gkzz** [renovatebot/renovate#16908] [Request Help] [answered]
Hello, all.
My question is as bellow;

- How can I find out the exact depName?
- For example, what is depName of `terraform-version`? I guess the answer is `hashicorp/terraform`, by reading the following code
  - https://github.com/renovatebot/renovate/blob/8bd77f747b3da59cc955b2fc4ba3301bcb680ea2/lib/modules/manager/terraform-version/extract.spec.ts#L22
  - depName of Terraform version in `.tool-versions` is also `hashicorp/terraform` ??
---

I've alread read the following, however I have difficulty to grasp what depName is:(

> depName: The name of the dependency being updated

Ref: https://docs.renovatebot.com/templates/#other-available-fields

> depName is essentially the "display name" while packageName is the actual package name used to do the lookup.

Ref: https://github.com/renovatebot/renovate/discussions/14885

Thanks always!!
I always get help from renovate, and your team!

**Comment by PhilipAbed** [comment, marked as answer]
dep name -> dependency name,  and it's the package name ,

in your case terraform is here 
https://registry.terraform.io/providers/hashicorp/terraform/1.0.2

but this file (`.terraform-version`) is special  because it includes only a version without the actual dependency name.

if you look at other files like `package.json` for example, you will see dependency name + the version range or the pinned version, which would make more sense.

we need the dependency name in order to get the package releases from the datasource and find newer version

**Reply by gkzz** [reply]
@PhilipAbed 
> dep name -> dependency name, and it's the package name ,
>
> in your case terraform is here
> https://registry.terraform.io/providers/hashicorp/terraform/1.0.2

Thanks for your quick response!!

I have one more question.

Node.js in .node-version or .tool-versions of depName is `nodejs/node`, because the Datasource as github-releases  is as follows;
https://github.com/nodejs/node/releases

Is that correct?

**Reply by rarkins** [reply]
Run in debug mode and look at the "packageFiles with updates" log message for a full list of dependencies and their depNames

**Reply by gkzz** [reply]
@rarkins 

> Run in debug mode and look at the "packageFiles with updates" log message for a full list of dependencies and their depNames

Thanks for your kindness! I got it:)
